hail/src/main/scala/is/hail/backend/BackendServer.scala

```diff
@@ -126,7 +143,7 @@ class BackendHttpHandler(backend: Backend) extends HttpHandler {
       exchange.sendResponseHeaders(200, response.length)
       using(exchange.getResponseBody())(_.write(response))
     } catch {
-      case t: Throwable =>
+      case NonFatal(t) =>
```

Comment:
Really appreciate this, not sure why we were catching _all_ throwables.

Reply:
Convenience I suppose. Easily done.